src/app/app.component.html

```diff
@@ -34,6 +34,38 @@ <h1 class="display-2 main-title" [class.mobile-title]="isMobile"><strong>My Holo
   </div>
 </section>
 
+<strong>
+  <div class="row">
+    <div class="col-3">
+      <a class="credits" target="_new" href="//twitter.com/ChenMorSays">
+        <div class="credits-pos-1"> Chen
```

Comment:
we shouldn't give class to every element if it should be unique use 'id' instead but this is not the way